app/controllers/concerns/css_id_concern.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module CssIdConcern
+  extend ActiveSupport::Concern
+
+  # :reek:UtilityFunction
+  def invalid_css_id?(css_id)
```

Comment:
Can we use "normalize" as we use in [user.rb](https://github.com/department-of-veterans-affairs/caseflow/blob/57e746e91b91c82c9b5f4edac343bde9688b68d9/app/models/user.rb#L391)? Upcasing does not necessarily mean the css_id will be valid.
```suggestion
  def non_normalized_css_id?(css_id)
```